### PR TITLE
Add email and address to bot

### PR DESCRIPTION
**Pull request by svilav**
* Updated `help` message
* Add `add`, `change`, and `email`/`address` (show)  commands for email and address
* Add necessary function to InputManager and AddressBook

@TetianaFilonenko plz check if this code aligns with all the other parts well. I'd be happy to improve this code.
@OF3000 plz check if you'd be comfortable working further with this code

### CODE REVIEW

File: console_bot.py [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
 
+
 def print_help():
-    """Function printing help messagefor bot."""
+    """Function printing help message for bot."""
     help_text = """
 Available commands:
   hello                          - Ask the bot how it can help you.
-  add [name] [phone]             - Adds a contact with the specified name and phone number.
+  add [name] [phone]             - Adds a contact with the specified name and phone number. Email and address can be added using separate commands.
```
Comment (TetianaFilonenko) [inline console_bot.py]: very informative 🔥

File: console_bot/address_book.py [hunk 2]
```diff
@@ -234,6 +234,48 @@ def get_next_week_birthdays(self):
 
         return get_birthdays_per_week(result)
 
+    def add_email(self, name, email):
+        if name in self.data:
+            is_valid, message = self.data[name].add_email(email)
+            return message
+        else:
+            return f"Contact {name} not found."
```
Comment (TetianaFilonenko) [inline console_bot/address_book.py]: Maybe we can expand a bit the message and ask to add contact first. wdyt?
```suggestion
            return f"Contact {name} not found. Add it first to the contact book"
```

File: console_bot/address_book.py [hunk 3]
```diff
@@ -234,6 +234,48 @@ def get_next_week_birthdays(self):
...
+
+    def add_address(self, name, address):
+        if name in self.data:
+            is_valid, message = self.data[name].add_address(address)
+            return message
+        else:
+            return f"Contact {name} not found."
+
+    def change_address(self, name, new_address):
+        if name in self.data:
```
Comment (TetianaFilonenko) [inline console_bot/address_book.py:286]: Technically change works as `add` if you call it on contact with empty address, which is fine

**Review by TetianaFilonenko — COMMENTED** [review]
Catch a bug
Contact name: tetiana; phones: 1234567890; emails: tetete@gmail.com; addresses: test
Enter a command: change-email teti invalid
Invalid email format: invalid
Enter a command: all
Contact name: teti; phones: 1234567890; addresses: test - here, email was replace with invalid, same for address

**Comment by svilav** [comment]
@TetianaFilonenko I fixed the bug, plz check. I decided to force the user to enter old value when changing as we hale lists of values and it is not clear what to change. I hope you are ok with that

**Review by TetianaFilonenko — APPROVED** [review]
Tested! works charmy
Great work 👍

**Pull request merged**